src/index.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 import App from './containers/App';
+import css from './style.css';
```

Comment:
The css and html import here are not necessary

Reply:
File-loader needs these imports to know which files are relevant